applications/endoscopy_tool_tracking/cpp/README.md

```diff
@@ -49,8 +49,7 @@ In your `build` directory, run the commands of your choice:
 
 * Using an AJA card
     ```bash
-    sed -i -e 's#^source:.*#source: aja#' applications/endoscopy_tool_tracking/cpp/endoscopy_tool_tracking.yaml
-    applications/endoscopy_tool_tracking/cpp/endoscopy_tool_tracking
+    ./run launch endoscopy_tool_tracking cpp --extra_args -capplications/endoscopy_tool_tracking/cpp/endoscopy_tool_tracking_aja_overlay.yaml
```

Comment:
I don't have AJA card at hand to test, could you please double check and confirm that the application will run and use overlay when executing this command and the one for Python? Thanks